snapcraft/plugins/v2/_ros.py

```diff
@@ -133,7 +133,7 @@ def stage_runtime_dependencies(part_install: str, ros_distro: str):
     # TODO: support python packages (only apt currently supported)
     apt_packages: Set[str] = set()
 
-    for pkg in catkin_packages.find_packages(".").values():
+    for pkg in catkin_packages.find_packages("$SNAPCRAFT_PART_BUILD_WORK").values():
```

Comment:
This isn't bash, you'll need to pass in the directory to the class.

Reply:
My bad, I actually didn't mean to push this since it is not necessary. I'll revert it.

Reply:
Reverted in 12087c9.